SyncUp.Data/Models/Tenancy/Tenant.cs

```diff
@@ -24,7 +25,41 @@ public class Tenant
     [ForeignKey(nameof(BannerImageId))]
     public Image? BannerImage { get; set; }
 
+    [Coalesce]
+    public async Task<ItemResult<Image>> UploadImageFile(AppDbContext db, [Inject] ImageService imageService, File file)
```

Comment:
Code here can be reduced if you just make one shared function and 2 lambda functions. 🤷‍♀️